modules/ocf_desktop/files/xsession/fix-displays

```diff
@@ -11,6 +10,20 @@ def get_monitors():
     return [line.split()[0] for line in lines if ' connected' in line]
 
 
+def is_daisy_chaining(monitors):
```

Comment:
Explain what "daisy chaining" is in the docstring

Reply:
Yeet, included it.